frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentStepper.tsx

```diff
@@ -24,11 +27,17 @@ export const PublicEnrollmentStepper = ({
     }
   };
 
-  const getDescription = (stepNumber: number) =>
-    t(`step.${PublicEnrollmentFormStep[stepNumber]}`);
+  const getDescription = (stepNumber: number) => {
+    const i =
+      !includePaymentStep && stepNumber >= PublicEnrollmentFormStep.Payment
```

Comment:
Syy miksi täällä verrataan Payment-steppiin ja yllä Done-steppiin liittyy sellaiseen, että näin voitaisiin teoriassa lisätä myös jotain steppejä näiden kahden väliin jos jostain syystä olis tarvis

Reply:
Oiskohan tässä selkeempi jos tekis toisen enumin tyyliin
```
type PublicEnrollmentFormStepQueued = Exclude<PublicEnrollmentFormStep, PublicEnrollmentFormStep.Payment>;
```

En kokeillu niin en osaa varmuudella sanoa onnistuisko noin.

Reply:
Vois melkeinpä kokeilla vielä tällaista sen sijaan että mergeää suoralta kädeltä pois

Reply:
En kyllä suoraan keksi mitä tälle tekisi, jos tekisi jotain. Ratkaisu on kieltämättä vähän hack / monimutkainen, mutta koskee sentään vain tätä yhtä komponenttia.